spec/javascript/utils/compiler/variables.test.js

```diff
@@ -67,9 +66,7 @@ describe("variables.js", () => {
         Modify Player Variable At Index(Event Player, variable11, Test);
         Chase Player Variable At Rate(Event Player, variable12, Test, ...);
         Chase Player Variable Over Time(Event Player, variable13, Test, ...);
-        For Player Variable(Event Player, variable14, 1) {
-          // Do something
-        }
+        For Player Variable(Event Player, variable14, 0, 1);
```

Comment:
Ditto. Please don't edit remove curly braces here. There are in the tests to check that they don't interfere :)

Reply:
maybe im missing something here but why test that it works when its the wrong syntax? you cant paste it into the game, its not even the right number of parameters

Reply:
We are not testing that it works in-game, we are testing that it compiles. The compiler is allowed to be dumb sometimes, specially for making tests shorter and more to the point.
Though I guess for this test the curly braces _are_ kind of dumb, especially when there haven't been reports of curly braces after actions breaking `getVariables` before.

I guess my main issue here is that changing this is out of scope for this PR.

Reply:
yeah fair enough, i just assumed it was a mistake but thats probably not something i should just assume 🙃

Reply:
Ah yes, that was definitely a mistake :D